### DISCUSSION TITLE
Add IP2Location.io

### DISCUSSION THREAD

**Discussion by ip2location-com** [avelino/awesome-go#5167] [Ideas]
Would it be possible to add the [IP2Location.io Go SDK](https://github.com/ip2location/ip2location-io-go) to your list?

It works with the [IP2Location.io API](https://www.ip2location.io/) that allows users to query an IP address to get its geolocation data. It also can query the WHOIS data for a domain name.

**Comment by avelino** [comment]
send PR pls
